### ISSUE TITLE
[BUG] when I use 'occlum mount /occlum/files' command to mount host file, it's no returns.

### ISSUE THREAD

**Issue by mapleshi** [occlum/occlum#1423]
# when I use 'occlum mount /occlum/files' command to mount host file, it's no return.
**debug infos:**
Enclave sign-tool: /opt/occlum/sgxsdk-tools/bin/x64/sgx_sign
Enclave sign-key: /opt/occlum/etc/template/Enclave.pem
SGX mode: HW
rm -rf /root/demos/java/occlum_instance/build
Building the initfs...
[+] Home dir is /root
[+] Open token file success! 
[+] Token file valid!
[+] Init Enclave Successful 30743375904770!
Generate the SEFS image successfully
Building new image...
[+] Home dir is /root
[+] Open token file success! 
[+] Token file valid!
[+] Init Enclave Successful 30764850741250!
Generate the SEFS image successfully
Building libOS...
Signing the enclave...
<EnclaveConfiguration>
    <ProdID>0</ProdID>
    <ISVSVN>0</ISVSVN>
    <StackMaxSize>1048576</StackMaxSize>
    <StackMinSize>1048576</StackMinSize>
    <HeapInitSize>201326592</HeapInitSize>
    <HeapMaxSize>1073741824</HeapMaxSize>
    <HeapMinSize>201326592</HeapMinSize>
    <TCSNum>192</TCSNum>
    <TCSMaxNum>4096</TCSMaxNum>
    <TCSMinPool>192</TCSMinPool>
    <TCSPolicy>1</TCSPolicy>
    <DisableDebug>0</DisableDebug>
    <MiscSelect>1</MiscSelect>
    <MiscMask>0x0</MiscMask>
    <ReservedMemMaxSize>5284823040</ReservedMemMaxSize>
    <ReservedMemMinSize>5284823040</ReservedMemMinSize>
    <ReservedMemInitSize>5284823040</ReservedMemInitSize>
    <ReservedMemExecutable>1</ReservedMemExecutable>
    <EnableKSS>0</EnableKSS>
    <ISVEXTPRODID_H>0</ISVEXTPRODID_H>
    <ISVEXTPRODID_L>0</ISVEXTPRODID_L>
    <ISVFAMILYID_H>0</ISVFAMILYID_H>
    <ISVFAMILYID_L>0</ISVFAMILYID_L>
    <PKRU>0</PKRU>
</EnclaveConfiguration>
tcs_num 192, tcs_max_num 4096, tcs_min_pool 192
The required memory is 5695696896B.
The required memory is 0x1537d7000, 5562204 KB.
Succeed.
Built the Occlum image and enclave successfully
occlum run JVM web app
Mount tool sign-tool: /opt/occlum/sgxsdk-tools/bin/x64/sgx_sign
Mount tool sign-key: /opt/occlum/etc/template/Enclave.pem
Mount point: /occlum/files
SGX mode: HW
Signing the mount tool...

<EnclaveConfiguration>
    <ProdID>0</ProdID>
    <ISVSVN>0</ISVSVN>
    <StackMaxSize>0x100000</StackMaxSize>
    <HeapMaxSize>0x20000000</HeapMaxSize>
    <TCSNum>1</TCSNum>
    <TCSPolicy>1</TCSPolicy>
    <DisableDebug>0</DisableDebug>
    <MiscSelect>0</MiscSelect>
    <MiscMask>0xFFFFFFFF</MiscMask>
</EnclaveConfiguration>
tcs_num 1, tcs_max_num 1, tcs_min_pool 1
The required memory is 538779648B.
The required memory is 0x201d2000, 526152 KB.
Succeed.
Start to mount the FS...
[+] Home dir is /root
[+] Open token file success! 
[+] Token file valid!
[+] Init Enclave Successful 2!

# Desc.
it's stop there(like up.) when code run to 'occlum mount /occlum/files'. Is there any help? very thank much.

# Code.
run_app() {
    jar_path=./target/teeAesEncrypt-0.0.1-SNAPSHOT.jar
    check_file_exist ${jar_path}
    jar_file=`basename "${jar_path}"`
    init_instance
    build_app
    echo -e "${BLUE}occlum run JVM web app${NC}"
    occlum mount /occlum/files 
    occlum run /usr/lib/jvm/java-11-alibaba-dragonwell/jre/bin/java -Xmx512m -XX:-UseCompressedOops -XX:MaxMetaspaceSize=64m -Dos.name=Linux -jar /usr/lib/spring/${jar_file}
}

# Environment

- HW: [SGX2] 
- OS: [Ubuntu20.04]
- Occlum version: [0.28.0]

**Comment by mapleshi** [comment]
it pause at 'Open token file success! [+] Token file valid! [+] Init Enclave Successful 2!'. Actually, there is still code left there that is not executed.

**Comment by liqinggd** [comment]
Yes, the mount command will not return. This is because the mount command will start a Linux FUSE FS server.

> The mount command will not return until the FUSE server is terminated in one of the two ways. The first one is to press ctrl+C. The second one is to use umount command. Both ways can terminate the server gracefully.

You can set it as a background process by adding the "&" symbol at the end of the command.
